keep/api/core/db.py

```diff
@@ -996,6 +996,7 @@ def get_last_alerts(tenant_id, provider_id=None, limit=1000) -> list[Alert]:
         alerts = []
         for alert, startedAt in alerts_with_start:
             alert.event["startedAt"] = str(startedAt)
+            alert.event["event_id"] = str(alert.id)
```

Comment:
I'd suggest the following so we don't override something that may be provided by the provider itself
```suggestion
            alert.event["keep_event_id"] = str(alert.id)
```

Reply:
done!